### ISSUE TITLE
additional improvement suggestion

### ISSUE THREAD

**Issue by paOol** [acceptbitcoincash/acceptbitcoincash#324]
instead of having all additional merchants being added as PR's, perhaps it would be worth a tiny little iput system instead.

Have a form people fill out with the necessary fields,
populate it into a database 
an admin curates the list and sees that all info is legit and not spam
"checks" off a field in the database row marking it as valid

the site displays only valid listings from the database.

this would not be too much work to implement on the front/backend and would rapidly allow more merchants being added versus manual commits and re-deployment each time.

**Comment by t0p0lin0** [comment]
This is a great idea, however the site does not have a backend/database. It is based on Jekyll which is a static site generator (actually quite perfect for this application), which builds the frontend based on data within the `_data/*.yml` files.

This is why we have created an issue template (technically, PRs aren't the only/required way to get content added). It isn't as friendly as a normal form, but the submissions so far have been pretty legible.

So basically, your idea has great merit, but due to the current structure of the project, it's not quite possible at this time. I suspect that as the site grows in popularity/submissions/total count of merchants listed, we'll need to refactor to a database-driven setup. Until then though, the team is pretty happy with the current workflow.

Thanks for the idea regardless!

**Comment by kenman345** [comment]
Note: The fact that Jekyll generates flat HTML files means its less prone to DDOS attacks. So changing to hit a backend for data would be less than ideal.
